backend/src/main/java/com/carffeine/carffeine/domain/chargestation/charger/ChargerState.java

```diff
@@ -24,6 +26,13 @@ public static ChargerState from(int input) {
                 .orElse(STATUS_UNKNOWN);
     }
 
+    public static ChargerState from(String input) {
+        return Arrays.stream(ChargerState.values())
+                .filter(it -> it.valueString.equals(input))
+                .findAny()
+                .orElse(STATUS_UNKNOWN);
+    }
+
```

Comment:
```suggestion
    public static ChargerState from(String input) {
        int num = Integer.parseInt(intput);
        return Arrays.stream(ChargerState.values())
                .filter(it -> it.value == num)
                .findAny()
                .orElse(STATUS_UNKNOWN);
    }

```

이런 방법은 어떤가요 그럼 관리할 필드가 하나 줄어서 나을 것 같아요